src/app/containers/Caption/__snapshots__/index.test.jsx.snap

```diff
@@ -85,10 +98,15 @@ exports[`should render caption text with example News offscreen text 1`] = `
   line-height: 1.125rem;
   font-family: ReithSans,Helvetica,Arial,sans-serif;
   font-weight: 400;
-  font-style: italic;
```

Comment:
Do we know why we're getting style changes here?  These snapshots don't seem to just be re-ordered - this `font-style` rule for example doesn't seem to be in the new snapshot at all

Reply:
I'm not sure. I think the matchSnapshotAsync function is now broken and is not snapshotting after dom changes. I don't think we should be worried though. I created the matchSnapshotAsync function to capture metadata of pages component when I was doing a metadata refactor. These snapshots are now not working but integration tests are testing page metadata so I think we should remove the matchSnapshotAsync function altogether now.

Reply:
I think this is to do with the 4.1.1 psammead-caption I've been working on. The package has been published to npm, but the Simorgh PR which uses the 4.1.1 component is currently in testing, so the updated snapshots aren't yet in latest. This would explain the removal of `font-style: italic;` and the replacement of many other caption styles.

Reply:
Ah could be @HarryVerhoef thanks for letting me know 🙂